core/random.go

```diff
@@ -146,33 +145,32 @@ func (r *Random) GetLastRandomness(coinbase common.Address, db *ethdb.Database,
 		return common.Hash{}, nil
 	}
 
-	randomness := common.Hash{}
-	randomnessSlice, err := (*db).Get(commitmentDbLocation(lastCommitment))
+	parentBlockHashBytes, err := (*db).Get(commitmentDbLocation(lastCommitment))
```

Comment:
How are we dealing with setting first randomness to 0 right now?

Reply:
If the smart contract does not contain the a previous commitment, the randomness of 0 is returned. In this case its value is common.Hash{}. https://github.com/celo-org/celo-blockchain/pull/325/files#diff-55aa515bdb3e537871a9f3140b9a65cdL146